Program.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
+﻿Console.WriteLine ("Assign numbers for a and b");
+int a = GetInput ("a = "), b = GetInput ("b = ");
+swap (ref a, ref b);
+Console.WriteLine ($"After swapping: a = {a}, b = {b}");
+
+int GetInput (string prompt) {
+   while (true) {
+      Console.Write (prompt);
+      if (int.TryParse (Console.ReadLine (), out int value)) return value;
+      else continue;
+   }
+}
+
+void swap (ref int a, ref int b) {

```

Comment:
Use expression bodied functions for single line methods
```suggestion
void swap (ref int a, ref int b) =>

```